src/imitation/scripts/train.py

```diff
@@ -209,6 +219,16 @@ def should_plot_now(epoch) -> bool:
 
     save(trainer, os.path.join(log_dir, "final"))
 
+    # Final evaluation of imitation policy.
```

Comment:
This seems useful. If we end up wanting more complex evaluation we might want to consider putting it into `imitation.scripts.policy_eval` to stick to UNIX philosophy of one-tool-one-job.